src/assets/images/IsomerLogoNoText.tsx

```diff
@@ -0,0 +1,106 @@
+export const IsomerLogoNoText = (
+  props: React.SVGProps<SVGSVGElement>
+): JSX.Element => {
```

Comment:
can't we have this as `.svg` under assets folder instead of as code?

Reply:
I think the main reason we do this is so that we can pass props to the svg if necessary!